config.yaml

```diff
@@ -1,40 +1,40 @@
-grpcPort: 50052 # GRPC port of the AuthZ service
-httpPort: 8081 # HTTP port of the Authz Service
-httpsPort: 8443 # HTTPS port of the AuthZ service
-logRequests: true # Request logging midddleware on/off - true/false
+#grpcPort: 50052 # GRPC port of the AuthZ service
```

Comment:
I'd add a comment on top and in the readme (section: Configuration) that the commented out values are default values and don't need to be explicitly set until another value is needed.